include/seqan3/std/span

```diff
@@ -54,6 +54,25 @@ struct is_std_array_impl<array<span_tp, span_sz>> : public true_type {};
 template <class span_tp>
 struct is_std_array : public is_std_array_impl<remove_cv_t<span_tp>> {};
 
+} // namespace std
+
+#if RANGE_V3_VERSION >= 901
```

Comment:
We require `>= 9.0.0`.  I guess you mean `>= 900`?
But then we wouldn't need this `#if` since `platform.hpp` already checks it

Reply:
no this was introduced between 9.0.1 and 10.0.0